packages/pharos-site/static/guidelines/checkbox.docs.mdx

```diff
@@ -32,35 +31,31 @@ import BestPractices from '../../src/components/statics/BestPractices.tsx';
   <BestPractices
     Do={
       <ul>
+        <li>Use checkboxes when users need to choose "yes" or "no" per option. </li>
         <li>
-          Use checkboxes when a user needs to choose "yes" or "no" per option (use a stand-alone
-          checkbox). For example, to toggle a setting on or off.
-        </li>
-        <li>
-          Checkboxes should work independently from each other: selecting one checkbox shouldn't
-          change the selection status of another checkbox in the list.
+          Checkboxes should work independently from each other, unless there are checkboxes to
+          control groups
         </li>
         <li>Checkboxes should always include a label.</li>
-        <li>Use a checkbox when accepting terms and conditions.</li>
         <li>Use checkboxes when choices are not mutually exclusive.</li>
-        <li>The list of choices should be in a logical order.</li>
+        <li>The list of choices should be in a logical order</li>
```

Comment:
There's inconsistent use of ending punctuation in this list—if one bullet in a list ends with punctuation, the other items in the list should as well for parallel structure.

I'm also not positive what this means—can it be clarified?

> Checkboxes should work independently from each other, unless there are checkboxes to control groups